person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, parent_permission: true, name: 'Unknown')
```

Comment:
- Hey, please, kindly remove the id from the parameter as you won't really be using it like it's an instance variable, instead, you want to give it a value like in this case `Random.rand(1..100)` as you want a unique value. please, update this.